src/hotspot/share/gc/shared/genCollectedHeap.cpp

```diff
@@ -591,6 +589,8 @@ void GenCollectedHeap::do_collection(bool           full,
     size_t young_prev_used = _young_gen->used();
     size_t old_prev_used = _old_gen->used();
 
```

Comment:
Upstream version doesn't have this empty line between `old_prev_used` and `prev_meta_sizes`.